### ISSUE TITLE
There is no validation for `consumer.request.timeout.ms` during consumer creation

### ISSUE THREAD

**Issue by serrss** [strimzi/strimzi-kafka-bridge#308]
It's possible to create a consumer with value `foo` for parameter `consumer.request.timeout.ms` via endpoint `POST /consumers/{groupid}`

An example of JSON:
```
{
  "name": "consumer1",
  "format": "binary",
  "auto.offset.reset": "earliest",
  "enable.auto.commit": "false",
  "fetch.min.bytes": "512",
  "consumer.request.timeout.ms": "foo" <------- need validation for this field
}
```

**Comment by serrss** [comment]
If look at the Kafka docs [1], parameter `request.timeout.ms` should be `int`





1. https://kafka.apache.org/documentation/

**Comment by ppatierno** [comment]
@serrss is there any log with errors and/or exceptions you see when trying this kind of operation?

**Comment by serrss** [comment]
@ppatierno there is no errors or exceptions in logs during this operation. And as we discussed before, it should be fixed in the scope of #318
